### PR TITLE
Add study.md (ruby-study)

### PR DESCRIPTION
**Pull request by NatasaPeic**

### CODE REVIEW

File: study.md [hunk 1]
```diff
@@ -19,38 +19,55 @@ Please write the code that would take the following integer `007` and return the
 string `"700"`
 
 ```ruby
-  # your answer here
+
+007.to_s.rjust(3, '0').reverse
```
Comment (RealWeeks) [inline study.md:23]: really nice work.  You caught the trick in the question

File: study.md [hunk 2]
```diff
@@ -19,38 +19,55 @@ Please write the code that would take the following integer `007` and return the
...
 array.
 
 ```ruby
-  # your answer here
+arr = [23,56,3,7]
+=> [23,56,3,7]
+arr.sort!
+=> [3, 7, 23, 56]
+array.reverse
+=> [56, 23, 7, 3]
```
Comment (RealWeeks) [inline study.md:38]: in .sort! you modify the original is the .reverse modified?

**Comment by RealWeeks** [comment]
Great job. Keep it up, nice job catching the "trick" question.

File: study.md (hunk 2 above)
Comment (NatasaPeic) [inline study.md:38]: Original array is modified, and reverse method is applied to the modified (original, not copy) array. So, reversed array is the modified one.

File: study.md (hunk 1 above)
Comment (NatasaPeic) [inline study.md:23]: Thank you. Google search helped. 

Resource: 
https://cbabhusal.wordpress.com/2014/09/30/adding-precedingleading-zeros-in-integers-in-rubyonrails/

Used for testing:
https://repl.it/languages/ruby

**Pull request closed without merging**